src/code/z_scene.c

```diff
@@ -441,8 +441,8 @@ void Scene_CommandAlternateHeaderList(PlayState* play, SceneCmd* cmd) {
 }
 
 void Scene_CommandCutsceneData(PlayState* play, SceneCmd* cmd) {
-    osSyncPrintf("\ngame_play->demo_play.data=[%x]", play->csCtx.segment);
-    play->csCtx.segment = SEGMENTED_TO_VIRTUAL(cmd->cutsceneData.data);
+    osSyncPrintf("\ngame_play->demo_play.data=[%x]", play->csCtx.scriptPtr);
+    play->csCtx.scriptPtr = SEGMENTED_TO_VIRTUAL(cmd->cutsceneData.data);
```

Comment:
I don't think having "Ptr" is needed?

Reply:
its a case where i felt like it might help. Cause its not the actual script itself. 
I also come from speedrunning where we have called this "the cutscene pointer" since the beginning of time. So my perspective is probably skewed.

Reply:
it also worth noting that there `sDebugCutsceneScript` and `gDebugCutsceneScriptPtr`, so the symmetry is nice. But this is overall a minor point, cause the debug thing is mostly irrelevant. 
Moreso just saying that there is precedent for treating the pointer and the script itself as different things

Reply:
fair, I can change it. I do feel lit makes more sense to keep in the context of `Cutscene_ProcessScript` and a few of the other functions that take the pointer as an arg and do pointer math on it. what are your thoughts

Reply:
Personally I like without the `Ptr`, and see it as analogous to not calling `Actor*` `actorPtr`, though this might be MM bias, where we already have a script system that just uses `script`.